src/components/navigation-bar/ui/side-navigation/SideNavigationList.tsx

```diff
@@ -58,7 +80,7 @@ const SideNavigationList: FC<SideNavigationListProps> = ({ onItemClick, items })
             },
           },
           TreeLabel: {
-            component: NavTreeLabel,
+            component: ({ ...props }) => <NavTreeLabel {...props} itemAs={itemAs} />,
```

Comment:
I would separate all overrides to decrease jsx and use like
```
overrides={{getOverrides(component)}}
```
What do you think about that?